releases/cosmwasm/2025-01-ITS-v1.1.0.md

```diff
@@ -27,8 +27,15 @@ Changes in this release:
 
 Upload new ITS Hub contract
 
+| environment | INIT_ADDRESSES    |  RUN_AS_ACCOUNT |
+| :-----: | :---: | :---: |
+| devnet-amplifier | axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj,axelar1zlr7e5qf3sz7yf890rkh9tcnu87234k6k7ytd9  | axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj   |
+| stagenet | axelar1pumrull7z8y5kc9q4azfrmcaxd8w0779kg6anm,axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj,axelar12qvsvse32cjyw60ztysd3v655aj5urqeup82ky    | axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj   |
+| testnet | axelar1uk66drc8t9hwnddnejjp92t22plup0xd036uc2,axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj,axelar12f2qn005d4vl03ssjq07quz6cja72w5ukuchv7   | axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj   |
+| mainnet | axelar1uk66drc8t9hwnddnejjp92t22plup0xd036uc2,axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj,axelar1nctnr9x0qexemeld5w7w752rmqdsqqv92dw9am   | axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj   |
```

Comment:
@talalashraf @milapsheth Please double check the keys for `INIT_ADDRESSES`.  I copied it over from previous ITS byte code upload